pysrc/scripts/evaluation.py

```diff
@@ -1,11 +1,13 @@
 import argparse
 import csv
+from dataclasses import dataclass
 import json
 import os
 import sys
 from typing import Literal, Optional
 
 import pandas as pd
+from pysrc.models.ind import IND, RankedIND
```

Comment:
Maybe we can use relative imports here aswell to be consistent with the rest of the project? but it's only a suggestion

Reply:
I've accidentally pushed the changes for this to #37, so they'll be introduced there.